standards/application/telemetry.md

```diff
@@ -0,0 +1,39 @@
+---
+title: TELEMETRY
+name: Waku v2 Telemetry
```

Comment:
I think we should stop using the `v2` term
```suggestion
name: Waku Telemetry
```

Reply:
That's more of a meta concern cc @jm-clius

Reply:
Indeed. We're certainly not consistent here yet, but:
- the _name_ of a new spec should no longer include "v2" (just call it "Waku Telemetry")
- the _title_ of specs should preferably be prefixed with `WAKU2-` (i.e. I propose `WAKU2-TELEMETRY` as title)

I've updated the contribution guidelines to reflect this: https://github.com/waku-org/specs/blob/master/README.md#contributions